crates/tauri/src/app.rs

```diff
@@ -1092,6 +1096,57 @@ impl<R: Runtime> App<R> {
     });
   }
 
+  /// Runs the application, returning the exit code to use.
```

Comment:
can we add a Platform-specific note (see set_device_event_filter above fe) for no mobile support here? A bit more obvious than the cfg.